storage/buckets.rb

```diff
@@ -199,6 +199,133 @@ def delete_bucket project_id:, bucket_name:
+  # [START storage_lock_retention_policy]
+  # project_id  = "Your Google Cloud project ID"
+  # bucket_name = "Name of your Google Cloud Storage bucket"
+
+  require "google/cloud/storage"
+
+  storage = Google::Cloud::Storage.new project_id: project_id
+  bucket  = storage.bucket bucket_name
+
+  # Warning: Once a retention policy is locked it cannot be unlocked
+  # and retention period can only be increased.
+  # Uses bucket.metageneration as a precondition
+  bucket.lock_retention_policy!
+
+  puts "Retention policy for #{bucket_name} is now locked"
+  puts "Retention policy effective as of #{bucket.retention_effective_at}"
+  # [END storage_lock_retention_policy]
+end
+
+def remove_retention_policy project_id:, bucket_name:
+  # [START storage_remove_retention_policy]
+  # project_id  = "Your Google Cloud project ID"
+  # bucket_name = "Name of your Google Cloud Storage bucket"
+
+  require "google/cloud/storage"
+
+  storage = Google::Cloud::Storage.new project_id: project_id
+  bucket  = storage.bucket bucket_name
+
+  if !bucket.retention_policy_locked?
+    bucket.retention_period = nil
+    puts "Retention period for #{bucket_name} has been removed."
+  else
+    puts "Policy is locked and retention period can't be removed."
+  end
+  # [END storage_remove_retention_policy]
+end
+
+def get_retention_policy project_id:, bucket_name:
+  # [START storage_get_retention_policy]
+  # project_id  = "Your Google Cloud project ID"
+  # bucket_name = "Name of your Google Cloud Storage bucket"
+
+  require "google/cloud/storage"
+
+  storage = Google::Cloud::Storage.new project_id: project_id
+  bucket  = storage.bucket bucket_name
+
+  puts "Retention policy:"
+  puts "period: #{bucket.retention_period}"
+  puts "effective time: #{bucket.retention_effective_at}"
+  puts "policy locked: #{bucket.retention_policy_locked?}"
+  # [END storage_get_retention_policy]
+end
+
+def enable_default_event_based_hold project_id:, bucket_name:
+  # [START storage_enable_default_event_based_hold]
+  # project_id  = "Your Google Cloud project ID"
+  # bucket_name = "Name of your Google Cloud Storage bucket"
+
+  require "google/cloud/storage"
+
+  storage = Google::Cloud::Storage.new project_id: project_id
+  bucket  = storage.bucket bucket_name
+
+  bucket.update do |b|
+    b.default_event_based_hold = true
+  end
+
+  puts "Default event-based hold was enabled for #{bucket_name}"
+  # [END storage_enable_default_event_based_hold]
+end
+
+def disable_default_event_based_hold project_id:, bucket_name:
+  # [START storage_disable_default_event_based_hold]
+  # project_id  = "Your Google Cloud project ID"
+  # bucket_name = "Name of your Google Cloud Storage bucket"
+
+  require "google/cloud/storage"
+
+  storage = Google::Cloud::Storage.new project_id: project_id
+  bucket  = storage.bucket bucket_name
+
+  bucket.update do |b|
+    b.default_event_based_hold = false
+  end
+
+  puts "Default event-based hold was disabled for #{bucket_name}"
```

Comment:
This sentence is missing a period at the end.

Reply:
What about: "Default event-based hold is now #{bucket.default_event_based_hold?} for #{bucket_name}."